task2.js

```diff
@@ -1,6 +1,16 @@
 const csvPath = './csv/node_mentoring_t1_2_input_example.csv';
 const csv = require('csvtojson');
+const fs = require('fs');
 
 csv().fromFile(csvPath).then((jsonObj) => {
-  console.log(jsonObj);
+//   console.log(jsonObj);
+//   console.log(typeof jsonObj);
+  if (jsonObj) {
+    jsonObj.forEach((value) => {
+      fs.appendFile('task2.txt', `${JSON.stringify(value)}\n`, (err) => {
```

Comment:
On the homework, one of the requirements is that you use the provided format for the text file.

Your text file is different in 3 ways:
1 - The object properties in the example are all lowercase and yours are capitalized. Ex: "Book" should be "book"
2 - The price property in the example is a number, your file is formatted as string. Ex: "12.99"
3 - In the example, there is no amount property, you file contains it.

Reply:
You can use the [Column parser option](https://github.com/Keyang/node-csvtojson#column-parser) from csvtojson to fix 2 and 3
And for 1 you can define your [own headers](https://github.com/Keyang/node-csvtojson#header-row).

Reply:
I use your solution to handle this part and the structure is corrent now.